21/hw-21.py

```diff
@@ -0,0 +1,4 @@
+import sqlite3
+
+connection = sqlite3.connect('homework-21.sqlite')
+# print(connection)
```

Comment:
nit: there will be removed all redundant prints or comments or other debugging tools before push in repo